Projects/Presentation/Sources/Home/EmployStatus/EmployStatusViewController.swift

```diff
@@ -89,6 +91,21 @@ public final class EmployStatusViewController: BaseViewController<EmployStatusVi
                 self?.chartView.setChartData(model: info)
             }
             .disposed(by: disposeBag)
+
+        [
+            classButton1,
+            classButton2,
+            classButton3,
+            classButton4
+        ]
+            .enumerated()
+            .forEach { classNumber, button in
+                button.rx.tap
+                    .map { classNumber + 1 }
+                    .bind(to: classButtonTapped)
+                    .disposed(by: disposeBag)
+            }
```

Comment:
classNumber가 아니라 index라고 하는게 맞는 것 같습니다

Reply:
> 아니면 굳이 enumerated()를 쓰지 말고 foreach에서 button에 접근 할 수 있는데 ClassButton에 상수를 만들어 classNumber를 가져올 수 있게 하면 좋을 것 같아요

선배님이 말씀하신 방식으로 처음 코드를 작성하였는데 주영 선배님이 추천하는 방식으로 바꾼게 이 코드입니다
혹시 어떤 방식이 조금 더 효율적일까요?

Reply:
음 엄연히 다른 코드라고 생각해요. 기존에 튜플로 매핑해둔 구문을 `enumerate`를 사용해 더 깔끔한 코드를 제안했어요
저는 `Projects/Presentation/Sources/Home/EmployStatus/Components/ClassButton.swift #line=25` 에 init으로 이미 classNumber를 받고 있으니 상수로 classNumber을 하나 갖고, 잇으면
```suggestion
            .forEach { button in
                button.rx.tap
                    .map { button.classNumber }
                    .bind(to: classButtonTapped)
                    .disposed(by: disposeBag)
            }
```
이런식으로 굳이 index(offset)을 사용하지 않고도 활용할 수 있어요. 이것도 가독성 차원에서 약간 의견을 낸 거 뿐이긴 하니 뭐가 더 나을지는 개인이 판단해보세요!

추가로 tag를 써서 mapping 하라는거랑도 거의 비슷한 내용입니다~ 한번 생각해봐요!